public/components/layout/SearchBar.vue

```diff
@@ -55,7 +55,13 @@ export default Vue.extend({
     },
 
     variables(n, o) {
-      if (n !== o) {
+      if (_.isEqual(n, o)) {
```

Comment:
ah fair, let me fix that and make sure it still works.

Reply:
Breaks of course. I think I'll just take the value check out because I need it updating every time those computes run, or at least every time language runs. The fun of nudging preact along from vue I guess.

Reply:
good catch, was able to simplify this to a basic equals for just the language and get enough updating.